src_plugins/enum/ppx_deriving_enum.cppo.ml

```diff
@@ -77,7 +63,9 @@ let mappings_of_type type_decl =
     match mappings with
     | (a, { txt=atxt; loc=aloc }) :: (b, { txt=btxt; loc=bloc }) :: _ when a = b ->
       let sigil = match kind with `Regular -> "" | `Polymorphic -> "`" in
-      let sub = [Location.errorf ~loc:bloc "Same as for %s%s" sigil btxt] in
+      let sub =
+        [Ocaml_common.Location.errorf
+          ~loc:bloc "Same as for %s%s" sigil btxt] in
```

Comment:
Note: we will have to document this small change patterns to third-party plugin authors.